massgov/mayflower
# DP-19331 Set the nav bar height consistent in any screen sizes

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/develop/docs/for-developers/changelog-instructions.md)


## Description

Set the height of the nav bar consistent regardless of the screen size.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-19331)

## Steps to Test


1. Go to the following pages:
        - http://localhost:3000/patterns/03-organisms-by-template-header-hamburger/03-organisms-by-template-header-hamburger.html
        - http://localhost:3000/patterns/05-pages-Homepage-hamburger/05-pages-Homepage-hamburger.html
        - http://localhost:3000/patterns/03-organisms-by-template-header-mixed/03-organisms-by-template-header-mixed.html
        - http://localhost:3000/patterns/05-pages-Homepage-mixed/05-pages-Homepage-mixed.html

2. Check the nav bar height in Computed by changing the screen width, and find it stays same.

<img width="1064" alt="Mass_Gov" src="https://user-images.githubusercontent.com/9633303/87071130-c67e6600-c1e7-11ea-8619-b869a42bb125.png">

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


*

#### @TODO


*

#### Today I learned...
